### PR TITLE
#162163234 Pagination support for articles

### PR DESCRIPTION
**Pull request by joshtrigger**
**What does this PR do ?**
it adds pagination support for all articles
Every page accomomdates 5 articles

**Screenshots**
image

**Relevant pivotal tracker stories**
#162163234

### CODE REVIEW

File: src/actions/PaginationAction.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import actionTypes from "./actionTypes";
+
+export const getMoreArticles = page => dispatch => {
+  return fetch(
+    `https://ah-backend-athena-staging.herokuapp.com/api/articles?limit=5&page=${page}`,
```
Comment (JaymesKat) [inline src/actions/PaginationAction.js]: use an environment variable for the host name.

File: src/actions/__tests__/PaginationAction.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe("pagination", () => {
+  afterEach(() => {
+    fetchMock.restore();
+  });
+  it("test fetching api", () => {
+    fetchMock.getOnce(
+      "https://ah-backend-athena-staging.herokuapp.com/api/articles?limit=5&page=1",
```
Comment (JaymesKat) [inline src/actions/__tests__/PaginationAction.spec.jsx]: use an env variable for the backend host name.

**Review by JaymesKat — CHANGES_REQUESTED** [review]

File: src/actions/PaginationAction.js (hunk 1 above)
Comment (joshtrigger) [reply src/actions/PaginationAction.js]: okay, let me work on it right away

**Review by bibangamba — CHANGES_REQUESTED** [review]
image

When I reach the end (no pages) and I hit next, it shows the above. same thing for hitting previous. Could you tweak the implementation to disable or hide the 'next' button when we are on the last page or hide the 'previous' button when we're on the first page? The work is good nonetheless.

**Comment by JaymesKat** [comment]
@joshtrigger I have come across a bug. After landing on the articles page and I press the `Previous` button, I get a blank page.
It also happens when I click `Next` while on the last page in the pagination. Is this something you can fix quickly?
pr-pagination

**Review by kafuuma — APPROVED** [review]
This looks good @joshtrigger 
I tested and seems to work weel

**Review by SokoPaulSokool — APPROVED** [review]

**Review by bibangamba — APPROVED** [review]

**Review by Nta1e — APPROVED** [review]
I have checked on the deployment and it works. Good work! As long as you resolve the merge conflicts

**Review by JaymesKat — APPROVED** [review]

**Review by kafuuma — APPROVED** [review]

**Pull request merged**